### PR TITLE
Added progress indicator to profile

### LINKED ISSUE #369: add wizard / progress UI to profile / enrollment forms
Instead of tabs, the profile / enrollment forms should have an indicator so users know which step they are on. 

There are no links to jump between forms. The only way to advance is to click the "save and continue" button. (we need to add a "Previous Step" button but there's no design yet)

Here is the most recent design:
step two - alt wizard 2

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #369
Rebased on #420

#### What's this PR do?
Removes tab links, uses SVG to display a profile indicator like in the issue mockup.

#### Where should the reviewer start?
static/js/util/util.js

#### How should this be manually tested?
Go to each page in the profile and verify that the progress display looks correct

#### Screenshots (if appropriate)
screenshot from 2016-05-27 13-47-52

### CODE REVIEW

**Comment by giocalitri** [comment]
can you rebase master?

File: static/js/util/util.js [hunk 1]
```diff
@@ -207,6 +207,118 @@ export function makeRunStatusDisplay(run) {
...
+    let colorScheme;
+    if (i < active) {
+      colorScheme = colors.completed;
+    } else if (i === active) {
+      colorScheme = colors.current;
+    } else {
+      colorScheme = colors.future;
+    }
+
+    // TODO
```
Comment (giocalitri) [inline static/js/util/util.js]: has this comment been left behind?

Comment (noisecapella) [inline static/js/util/util.js]: Yes, I'll remove it

File: static/js/util/util.js [hunk 2]
```diff
@@ -207,6 +207,118 @@ export function makeRunStatusDisplay(run) {
...
+    const x1 = paddingX + radius + circleDistance * i;
+    const x2 = paddingX + radius + circleDistance * (i + 1);
```
Comment (giocalitri) [inline static/js/util/util.js]: can you please give better names to these constants?

**Comment by giocalitri** [comment]
just one minor comment left, then 👍

**Pull request merged**